magellan-journey-sample/src/main/java/com/wealthfront/magellan/sample/MainActivity.kt

```diff
@@ -1,21 +1,61 @@
 package com.wealthfront.magellan.sample
 
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
 import androidx.appcompat.app.AppCompatActivity
 import com.wealthfront.magellan.lifecycle.setContentScreen
+import com.wealthfront.magellan.sample.App.Provider.appComponent
+import com.wealthfront.magellan.sample.menu.MenuProvider
+import javax.inject.Inject
 
 class MainActivity : AppCompatActivity() {
 
-  private val expeditionProvider = ExpeditionProvider
+  @Inject lateinit var expedition: Expedition
+  @Inject lateinit var menuProvider: MenuProvider
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentScreen(expeditionProvider.expedition)
+    appComponent.inject(this)
+    setContentScreen(expedition)
+    initActionBar()
+  }
+
+  private fun initActionBar() {
+    val actionBar = super.getSupportActionBar()!!
```

Comment:
Do we want to use a `Toolbar` here, since that's google's recommended way of doing this?

Reply:
uh. Im just gonna leave this as is since its just a sample..